package.json

```diff
@@ -467,7 +467,7 @@
         "test": "./scripts/test.sh",
         "package": "vsce package",
         "update-snapshot": "./scripts/test.sh -u",
-        "vscode:prepublish": "./scripts/build.sh",
+        "vscode:prepublish": "./scripts/build.sh && node esbuild.js",
```

Comment:
Maybe we want to merge `build.sh` and `esbuild.js` into a single `build.js`? That would potentially keep our buld more straightforward

Reply:
I just made a package.sh for doing the dependent tasks instead of doing it in prepublish. To me prepublish is some chore after build package and just befre the publishing (like login, checking credentials).